akka-operator/templates/040-deployment.yaml

```diff
@@ -31,7 +116,22 @@ spec:
         - name: {{ .Chart.Name }}
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
+          # No provider name found
+          {{- if (not .Values.provider.name) }}
+          image: {{ fail "A valid .Values.provider.name entry required! Valid values: gcp, aws" }}
```

Comment:
Due to "how the gcp marketplace works" we don't have a "default" image for GCP (i.e. is dynamically configured by the deployer), I'm sorry to waste this work, but I think we can roll-back and keep AWS as default.

Reply:
@andreaTP removed the check. Could u please take another look ?

Reply:
@andreaTP Now that I made this change, how will it work if the `image:` is set to the AWS one ? We will have a helm chart that will have `image:` set to the AWS repository - right ?

Reply:
I would go for this solution, maybe let's wait for a second opinion by @RayRoestenburg or @patriknw